xapian-core/include/xapian/weight.h

```diff
@@ -467,6 +467,33 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
        tfidf weight. */
     std::string normalizations;
 
+  public:
+    enum class WDF_NORM: int {
+	NONE = 1,
```

Comment:
Is there a reason not to start at `0`?

Reply:
I wasn't sure that 0 will be fine during serialise(). ( \0 may cause problem.)
Also I looked at other enums defined in weight class .They were also beginning form 1 .
So I chose 1 instead of 0.